developer_guides/topology2/topology2.rst

```diff
@@ -1332,6 +1332,51 @@ You can use the ``-P`` switch to convert a 2.0 configuration file to the 1.0 con
 
    alsatplg <-D args=values> -P input.conf -o output.conf
 
+Split topologies
+****************
+
+Linux kernel can load multiple topologies, a topology for a single function.
+This feature is useful when a single device is disabled in specific productuds. To achieve this, you need to split
+the topology into multiple tplg files. The split topology files should be named as follows:
+
+.. code-block:: bash
+
+        sof-<platform>-<function>-id<BE id number>.tplg
+
+Where <platform> should be the platform name like mtl, lnl, etc.
+
+Where <function> should be one of
+
+.. code-block:: bash
+
+        sdca-jack
+        sdca-<n>amp where n is the amp link numbers
+        sdca-mic
+        dmic-<n>ch where n is the channel numbers. Currently, only 2ch and 4ch are supported.
+        sdca-hdmi
+
+
+For example
+
+.. code-block:: bash
+
+        sof-arl-sdca-2amp-id2.tplg
```

Comment:
so in topology it is better to ship sof-arl-sdca-2amp-id[1-256].tplg to cover pretty much any place the 2amp can be?

Reply:
Not necessary, the BE id of amp is fixed to 2 in the machine driver. Unless there is a special case, otherwise, `-id2` is good enough.